app/src/main/java/org/p2p/wallet/sell/ui/error/SellErrorFragment.kt

```diff
@@ -13,28 +13,43 @@ import org.p2p.wallet.utils.popBackStack
 import org.p2p.wallet.utils.replaceFragment
 import org.p2p.wallet.utils.viewbinding.viewBinding
 import org.p2p.wallet.utils.withArgs
+import java.math.BigDecimal
 
 private const val ARG_ERROR_STATE = "ARG_ERROR_STATE"
+private const val ARG_MIN_AMOUNT = "ARG_MIN_AMOUNT"
 
 class SellErrorFragment : BaseFragment(R.layout.fragment_sell_error) {
 
     companion object {
-        fun create(errorState: SellScreenError): SellErrorFragment =
+        fun create(
+            errorState: SellScreenError,
```

Comment:
you can pass minAmount to particular SellScreenError to couple it more

Reply:
I think, we need here better abstraction, cause there is no way move minAmount to enum class, it was predefined with default values )